### PR TITLE
Created flow definitions for React Native Maps

### PR DESCRIPTION
**Pull request by danieljvdm**

### CODE REVIEW

**Comment by danieljvdm** [comment]
realized directories are wrong - fixing now

File: definitions/npm/react-native-maps_v0.16.x/flow_v0.53.x/react-native-maps_v0.16.x.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
+declare module 'react-native-maps' {
```
[the remark this answers is not in the document]
Comment (danieljvdm) [reply definitions/npm/react-native-maps_v0.16.x/flow_v0.53.x/react-native-maps_v0.16.x.js]: Great thanks!

[the remark this answers is not in the document]
Comment (danieljvdm) [reply definitions/npm/react-native-maps_v0.16.x/flow_v0.53.x/react-native-maps_v0.16.x.js]: Ok, fixed that now. Though I'm not sure what's the issue with the Event extension - it all works for me locally.

[the remark this answers is not in the document]
Comment (GAntoine) [reply definitions/npm/react-native-maps_v0.16.x/flow_v0.53.x/react-native-maps_v0.16.x.js]: Do the tests succeed locally?

[the remark this answers is not in the document]
Comment (danieljvdm) [reply definitions/npm/react-native-maps_v0.16.x/flow_v0.53.x/react-native-maps_v0.16.x.js]: `All tests passed!`

[the remark this answers is not in the document]
Comment (GAntoine) [reply definitions/npm/react-native-maps_v0.16.x/flow_v0.53.x/react-native-maps_v0.16.x.js]: Well I'm stumped. Let me find someone who can give us a hand...

[the remark this answers is not in the document]
Comment (danieljvdm) [reply definitions/npm/react-native-maps_v0.16.x/flow_v0.53.x/react-native-maps_v0.16.x.js]: Weird! That sounds good, thanks!

**Comment by danieljvdm** [comment]
Any news on why this is failing but succeeding locally?

**Comment by danieljvdm** [comment]
?

**Comment by GAntoine** [comment]
What version of flow are you using locally? This seems to be failing only on flow versions 53+.

**Pull request merged**